tests/opensource_demo_orangehrmlive/admin/job-categories/test_delete_category.py

```diff
@@ -0,0 +1,259 @@
+import json
+import pytest
+import requests
+from src.assertions.common_assertions import *
+from src.assertions.update_category_assertions import *
+from src.utils.loggers_helpers import log_request_response
+from src.assertions.delete_category_assertions import assert_category_not_exists, assert_deleted_category_ids, assert_id_deleted_once
```

Comment:
Podrias utilizar directo asterisco
```suggestion
from src.assertions.delete_category_assertions import *
```